src/main/java/picard/cmdline/StandardOptionDefinitions.java

```diff
@@ -46,4 +46,5 @@ public class StandardOptionDefinitions {
     public static final String MINIMUM_LOD_SHORT_NAME = "LOD";
     public static final String SORT_ORDER_SHORT_NAME = "SO";
     public static final String USE_ORIGINAL_QUALITIES_SHORT_NAME = "OQ";
+    public static final String USE_BQ_FOR_BASE_QUALITIES_SHORT_NAME = "UBQ";
```

Comment:
hopefully you won't need this after updating the tool to take the tag from the user rather than using "BQ"